## ▸ Main Studies

---------------------------------------


<p float="left">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_fwp.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/inferloop_logo.jpg" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_fitdevo.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_scmagic.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_vector.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_beer.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_phenopro.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_sprint.png" height="60">
</p>


  * Feature-weight based measurement of cancerous transcriptome using cohort-wide and sample-specific information, **Cellular Oncology, 2023**, {[code](https://github.com/jumphone/FWP), in press}, corresponding author

  * InferLoop: leveraging single-cell chromatin accessibility for the signal of chromatin loop, **Briefings in Bioinformatics, 2023**, {[code](https://github.com/jumphone/InferLoop), [paper](https://doi.org/10.1093/bib/bbad166)}, first & co-corresponding author

  * FitDevo: accurate inference of single-cell developmental potential using sample-specific gene weight, **Briefings in Bioinformatics, 2022**, {[code](https://github.com/jumphone/FitDevo), [paper](https://doi.org/10.1093/bib/bbac293)}, first & co-corresponding author

  * scMAGIC: accurately annotating single cells using two rounds of reference-based classification, **Nucleic Acids Research, 2022**, {[code](https://github.com/TianLab-Bioinfo/scMAGIC), [paper](https://doi.org/10.1093/nar/gkab1275)}, co-first author
  
  * Unsupervised Inference of Developmental Directions for Single Cells Using VECTOR, **Cell Reports, 2020**, {[code](https://github.com/jumphone/Vector), [paper](https://doi.org/10.1016/j.celrep.2020.108069)}, first author
  
  * A novel approach to remove the batch effect of single-cell data, **Cell Discovery, 2019**, {[code](https://github.com/jumphone/BEER), [paper](https://doi.org/10.1038/s41421-019-0114-x)}, first author
   
  * PhenoPro: a novel toolkit for assisting in the diagnosis of Mendelian disease, **Bioinformatics, 2019**, {[code](https://github.com/jumphone/PhenoPro), [paper](https://doi.org/10.1093/bioinformatics/btz100)}, co-first author

  * SPRINT: an SNP-free toolkit for identifying RNA editing sites, **Bioinformatics, 2017**, {[code](https://github.com/jumphone/SPRINT), [paper](https://doi.org/10.1093/bioinformatics/btx473)}, first author

<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ Collaborations

---------------------------------------

<img src="https://fzhang.bioinfo-lab.com/img/colla_logo.png" width="200" height="50">

  * Kinase-independent role of mTOR and on-/off-target effects of an mTOR kinase inhibitor, **Leukemia, 2023**, [paper](https://doi.org/10.1038/s41375-023-01987-w)
  
  * Loss of phosphatase CTDNEP1 potentiates aggressive medulloblastoma by triggering MYC amplification and genomic instability, **Nature Communications, 2023**, [paper](https://doi.org/10.1038/s41467-023-36400-8)
  
  * Inducible lncRNA transgenic mice reveal continual role of HOTAIR in promoting breast cancer metastasis, **Elife, 2022**, [paper](https://doi.org/10.7554/elife.79126)
  
  * Human fetal cerebellar cell atlas informs medulloblastoma origin and oncogenesis, **Nature, 2022**, [paper](https://doi.org/10.1038/s41586-022-05487-2)

  * Single-cell multiomics identifies clinically relevant mesenchymal stem-like cells and key regulators for MPNST malignancy, **Science Advances, 2022**, [paper](https://doi.org/10.1126/sciadv.abo5442)
  
  * Conserved and Distinct Functions of the Autism-Related Chromatin Remodeler CHD8 in Embryonic and Adult Forebrain Neurogenesis, **Journal of Neuroscience, 2022**, [paper](https://doi.org/10.1523/JNEUROSCI.2400-21.2022)
  
  * CDC42 controlled apical-basal polarity regulates intestinal stem cell to transit amplifying cell fate transition via YAP-EGF-mTOR signaling, **Cell Reports, 2022**, [paper](https://doi.org/10.1016/j.celrep.2021.110009)

  * OLIG2 maintenance is not essential for diffuse intrinsic pontine glioma cell line growth but regulates tumor phenotypes, **Neuro-Oncology, 2021**, [paper](https://doi.org/10.1093/neuonc/noab016)

  * Single-cell transcriptomic analysis reveals a hepatic stellate cell-activation roadmap and myofibroblast origin during liver fibrosis, **Hepatology, 2021**, [paper](https://doi.org/10.1002/hep.31987)

  * Adaptive responses to mTOR gene targeting in hematopoietic stem cells reveal a proliferative mechanism evasive to mTOR inhibition, **Proceedings of the National Academy of Sciences of the United States of America, 2020**, [paper](https://www.pnas.org/content/118/1/e2020102118)
 
  * CTCF-mediated chromatin looping in EGR2 regulation and SUZ12 recruitment critical for peripheral myelination and repair, **Nature Communications, 2020**, [paper](https://doi.org/10.1038/s41467-020-17955-2)

  * The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens, **Genome Biology, 2019**, [paper](https://doi.org/10.1186/s13059-019-1835-8)

  * Single-Cell Transcriptomics in Medulloblastoma Reveals Tumor-Initiating Progenitors and Oncogenic Cascades during Tumorigenesis and Relapse, **Cancer Cell, 2019**, [paper](https://doi.org/10.1016/j.ccell.2019.07.009)

  * Single-cell Transcriptomics Uncovers Glial Progenitor Diversity and Cell Fate Determinants during Development and Gliomagenesis, **Cell Stem Cell, 2019**, [paper](https://doi.org/10.1016/j.stem.2019.03.006)

  * Dual requirement of CHD8 for chromatin landscape establishment and histone methyltransferase recruitment to promote CNS myelination and repair, **Developmental Cell, 2018**, [paper](https://doi.org/10.1016/j.devcel.2018.05.022)

  * Genome-wide identification of regulatory sequences undergoing accelerated evolution in the human genome, **Molecular Biology and Evolution, 2016**, [paper](https://doi.org/10.1093/molbev/msw128)

<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ Experience & Education

---------------------------------------

&nbsp;&nbsp;&nbsp;&nbsp;<img src="https://fzhang.bioinfo-lab.com/img/jiaotong_logo.png" width="150">

* **Research Assistant Professor (助理研究员)**, Department of Histoembryology, Genetics and Developmental Biology, Shanghai Jiao Tong University School of Medicine, China, **2020-present**

* **Graduate Assistant & Visiting Scholar**, Cincinnati Children's Hospital Medical Center, USA, **2018-2020**. Collaborator: [Prof. Qing Richard Lu](https://www.cincinnatichildrens.org/bio/l/qing-richard-lu) and [Prof. Yi Zheng](https://www.cincinnatichildrens.org/bio/z/yi-zheng)

* **Teaching Asistant**, Biostatistics: Theories and Applications, Fudan University, China, **2015-2016**

* **PhD**, Biostatistics, School of Life Sciences, Fudan University, China, **2015-2020**. Supervisor: [Prof. Weidong Tian](https://life.fudan.edu.cn/9f/60/c31283a368480/page.htm)

* **Summer School**, University of California, Los Angeles, USA, **2013**. Introduction to Biostatistics. Instructor: Prof. Gjertson, D.W.

* **BSc**, School of Mathmatics, Shandong University, China, **2011-2015** 

<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ Contact

---------------------------------------

&nbsp;&nbsp;&nbsp;&nbsp;<img src="https://fzhang.bioinfo-lab.com/img/email_logo.png" width="200">

* **No.280 South Chong Qing Road, Shanghai, China, 200025**

* **fzhang@shsmu.edu.cn**


