<p float="left">
<img src="https://fzhang.bioinfo-lab.com/img/biuh_logo.jpg" height="60">
</p>

### **BiUH, CS, Associate Professor**

---------------------------------------


**Work History**: Research Associate Professor in Shanghai Jiao Tong University School of Medicine (Collaborator: [Prof Lingjie Li](https://www.shsmu.edu.cn/cbmsen/info/1072/1301.htm)). Visiting Scholar at Cincinnati Children's Hospital in the United States (Collaborator: [Prof Qing Richard Lu](https://www.cincinnatichildrens.org/bio/l/qing-richard-lu) and [Prof Yi Zheng](https://www.cincinnatichildrens.org/bio/z/yi-zheng)) 

**Education**: PhD, Department of Computational Biology at the School of Life Sciences, Fudan University (Supervisor: [Prof Weidong Tian](https://life.fudan.edu.cn/9f/60/c31283a368480/page.htm)
). BSc, School of Mathmatics, Shandong University. After graduating with a BSc, Feng was directly admitted to a Ph.D. program

**Research**: Feng's main research area is the development and innovative application of algorithms for single-cell and spatial multi-omics data, with years of experience in algorithm development and data analysis. Feng has led projects funded by the National Natural Science Foundation of China and the Shanghai Sailing Program (sponsored by the Science and Technology Commission of Shanghai Municipality) and participated in multiple national and provincial projects. Feng's main studies in algorithm development has been published in journals such as [Cell Reports](https://doi.org/10.1016/j.celrep.2021.110009), [Cell Discovery](https://doi.org/10.1038/s41421-019-0114-x), and [Brief Bioinform](https://doi.org/10.1093/bib/bbad166), whereas his collaborative research has been published in highly reputable journals such as [Cancer Cell](https://doi.org/10.1016/j.ccell.2019.07.009), [Cell Stem Cell](https://doi.org/10.1016/j.stem.2019.03.006), and [PNAS](https://www.pnas.org/content/118/1/e2020102118).


<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ Research & Publication

---------------------------------------

&nbsp;&nbsp;&nbsp;&nbsp;

  * Ultrasmall Cu2I2 nanoclusters trigger metabolic-epigenetic reprogramming and endogenous antioxidant systems for alleviating osteoarthritis, **Chemical Engineering Journal**, 2024.10, [paper](https://doi.org/10.1016/j.cej.2024.154568)
  
  * ARID1A loss promotes RNA editing of CDK13 in an ADAR1-dependent manner, **BMC Biology**, 2024.06, [paper](https://doi.org/10.1186/s12915-024-01927-9)

  * Long-read sequencing reveals alternative splicing-driven, shared immunogenic neoepitopes regardless SF3B1 status in uveal melanoma, **Cancer Immunology Research**, 2023.12, [paper](https://doi.org/10.1158/2326-6066.cir-23-0083)

  * ★ Quantitative analysis of the developmental potential of cells and tissues based on evolutionary conservation of genes and regulatory regions, **Journal of Shanghai Jiao Tong University (Medical Science)**, 2023.11, [paper](https://xuebao.shsmu.edu.cn/CN/10.3969/j.issn.1674-8115.2023.11.006), co-corresponding author

  * ★  Feature-weight based measurement of cancerous transcriptome using cohort-wide and sample-specific information, **Cellular Oncology**, 2023.10, {[code](https://github.com/jumphone/FWP), [paper](https://doi.org/10.1007/s13402-023-00879-6), [full-text](https://rdcu.be/dn9I1)}, corresponding author
  
  * Kinase-independent role of mTOR and on-/off-target effects of an mTOR kinase inhibitor, **Leukemia**, 2023.08, [paper](https://doi.org/10.1038/s41375-023-01987-w)

  * ★ InferLoop: leveraging single-cell chromatin accessibility for the signal of chromatin loop, **Briefings in Bioinformatics**, 2023.05, {[code](https://github.com/jumphone/InferLoop), [paper](https://doi.org/10.1093/bib/bbad166)}, first & co-corresponding author
  
  * Loss of phosphatase CTDNEP1 potentiates aggressive medulloblastoma by triggering MYC amplification and genomic instability, **Nature Communications**, 2023.02, [paper](https://doi.org/10.1038/s41467-023-36400-8)
  
  * Inducible lncRNA transgenic mice reveal continual role of HOTAIR in promoting breast cancer metastasis, **Elife**, 2022.12, [paper](https://doi.org/10.7554/elife.79126)
  
  * Single-cell multiomics identifies clinically relevant mesenchymal stem-like cells and key regulators for MPNST malignancy, **Science Advances**, 2022.11, [paper](https://doi.org/10.1126/sciadv.abo5442)
  
  * Conserved and Distinct Functions of the Autism-Related Chromatin Remodeler CHD8 in Embryonic and Adult Forebrain Neurogenesis, **Journal of Neuroscience**, 2022.11, [paper](https://doi.org/10.1523/JNEUROSCI.2400-21.2022)

  * ★ FitDevo: accurate inference of single-cell developmental potential using sample-specific gene weight, **Briefings in Bioinformatics**, 2022.09, {[code](https://github.com/jumphone/FitDevo), [paper](https://doi.org/10.1093/bib/bbac293)}, first & co-corresponding author
  
  * CDC42 controlled apical-basal polarity regulates intestinal stem cell to transit amplifying cell fate transition via YAP-EGF-mTOR signaling, **Cell Reports**, 2022.06, [paper](https://doi.org/10.1016/j.celrep.2021.110009)

  * ★ scMAGIC: accurately annotating single cells using two rounds of reference-based classification, **Nucleic Acids Research**, 2022.05, {[code](https://github.com/TianLab-Bioinfo/scMAGIC), [paper](https://doi.org/10.1093/nar/gkab1275)}, co-first author

  * OLIG2 maintenance is not essential for diffuse intrinsic pontine glioma cell line growth but regulates tumor phenotypes, **Neuro-Oncology**, 2021.11, [paper](https://doi.org/10.1093/neuonc/noab016)

  * Single-cell transcriptomic analysis reveals a hepatic stellate cell-activation roadmap and myofibroblast origin during liver fibrosis, **Hepatology**, 2021.11, [paper](https://doi.org/10.1002/hep.31987)

  * Adaptive responses to mTOR gene targeting in hematopoietic stem cells reveal a proliferative mechanism evasive to mTOR inhibition, **Proceedings of the National Academy of Sciences of the United States of America**, 2020.12, [paper](https://www.pnas.org/content/118/1/e2020102118)

  * ★ Unsupervised Inference of Developmental Directions for Single Cells Using VECTOR, **Cell Reports**, 2020.08, {[code](https://github.com/jumphone/Vector), [paper](https://doi.org/10.1016/j.celrep.2020.108069)}, first author
 
  * CTCF-mediated chromatin looping in EGR2 regulation and SUZ12 recruitment critical for peripheral myelination and repair, **Nature Communications**, 2020.08, [paper](https://doi.org/10.1038/s41467-020-17955-2)

  * The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens, **Genome Biology**, 2019.11, [paper](https://doi.org/10.1186/s13059-019-1835-8)

  * ★ A novel approach to remove the batch effect of single-cell data, **Cell Discovery**, 2019.09, {[code](https://github.com/jumphone/BEER), [paper](https://doi.org/10.1038/s41421-019-0114-x)}, first author

  * Single-Cell Transcriptomics in Medulloblastoma Reveals Tumor-Initiating Progenitors and Oncogenic Cascades during Tumorigenesis and Relapse, **Cancer Cell**, 2019.07, [paper](https://doi.org/10.1016/j.ccell.2019.07.009)

  * Single-cell Transcriptomics Uncovers Glial Progenitor Diversity and Cell Fate Determinants during Development and Gliomagenesis, **Cell Stem Cell**, 2019.03, [paper](https://doi.org/10.1016/j.stem.2019.03.006)

  * ★ PhenoPro: a novel toolkit for assisting in the diagnosis of Mendelian disease, **Bioinformatics**, 2019.02, {[code](https://github.com/jumphone/PhenoPro), [paper](https://doi.org/10.1093/bioinformatics/btz100)}, co-first author

  * Dual requirement of CHD8 for chromatin landscape establishment and histone methyltransferase recruitment to promote CNS myelination and repair, **Developmental Cell**, 2018.05, [paper](https://doi.org/10.1016/j.devcel.2018.05.022)

  * ★ SPRINT: an SNP-free toolkit for identifying RNA editing sites, **Bioinformatics**, 2017.07, {[code](https://github.com/jumphone/SPRINT), [paper](https://doi.org/10.1093/bioinformatics/btx473)}, first author
    
  * Genome-wide identification of regulatory sequences undergoing accelerated evolution in the human genome, **Molecular Biology and Evolution**, 2016.07, [paper](https://doi.org/10.1093/molbev/msw128)

<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ Work & Education

---------------------------------------

&nbsp;&nbsp;&nbsp;&nbsp;

* **Associate Professor**, Computer Science, Hainan Bielefeld University of Applied Sciences, China, **2025.02-present**

* **Research Associate Professor**, Department of Histoembryology, Genetics and Developmental Biology, Shanghai Jiao Tong University School of Medicine, China, **2024.01-2025.01**

* **Research Assistant Professor**, Department of Histoembryology, Genetics and Developmental Biology, Shanghai Jiao Tong University School of Medicine, China, **2020.10-2023.12**

* **Graduate Assistant & Visiting Scholar**, Cincinnati Children's Hospital Medical Center, USA, **2018.01-2020.01**. Collaborator: [Prof. Qing Richard Lu](https://www.cincinnatichildrens.org/bio/l/qing-richard-lu) and [Prof. Yi Zheng](https://www.cincinnatichildrens.org/bio/z/yi-zheng)

* **PhD**, Biostatistics, School of Life Sciences, Fudan University, China, **2015.09-2020.06**. Supervisor: [Prof. Weidong Tian](https://life.fudan.edu.cn/9f/60/c31283a368480/page.htm)

* **Summer School**, University of California, Los Angeles, USA, **2013.08-2013.09**. Introduction to Biostatistics. Instructor: Prof. Gjertson, D.W.

* **BSc**, School of Mathmatics, Shandong University, China, **2011.09-2015.06** 

<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ Contact

---------------------------------------

&nbsp;&nbsp;&nbsp;&nbsp;<img src="https://fzhang.bioinfo-lab.com/img/email_logo.png" width="200">

* Comming Soon




